main.go

```diff
@@ -18,10 +19,10 @@ type config struct {
 func parseConfig() (config, error) {
 	cfg := config{}
 	pflag.StringVarP(&cfg.Addr, "addr", "a", ":9210", "Address to listen on.")
-	pflag.StringVarP(&cfg.Netatmo.ClientID, "client-id", "i", "", "Client ID for NetAtmo app.")
-	pflag.StringVarP(&cfg.Netatmo.ClientSecret, "client-secret", "s", "", "Client secret for NetAtmo app.")
-	pflag.StringVarP(&cfg.Netatmo.Username, "username", "u", "", "Username of NetAtmo account.")
-	pflag.StringVarP(&cfg.Netatmo.Password, "password", "p", "", "Password of NetAtmo account.")
+	pflag.StringVarP(&cfg.Netatmo.ClientID, "client-id", "i", os.Getenv("ClientID"), "Client ID for NetAtmo app.")
```

Comment:
Environment variables are upper-case traditionally. How about these instead?

- `NETATMO_CLIENT_ID`
- `NETATMO_CLIENT_SECRET`
- `NETATMO_USERNAME`
- `NETATMO_PASSWORD`

Reply:
Another note: Usually environment variables have the highest priority, which is not the case here. Could you change the code so that environment variables take precedence over flags? Or do you think the current priority is better?

Reply:
I agree that usually env vars have higher priority, but I didn't want to pull in another external dependency or make the code overly complex as I don't think most users will use either env vars or command line arguments but not both.